extension/src/panel.ts

```diff
@@ -88,7 +85,7 @@ export class HistoryDebuggerPanel {
   private constructor(
     protected readonly extensionUri: vscode.Uri,
     private readonly secretStorage: vscode.SecretStorage,
-    protected readonly httpServerUrl: string,
+    protected readonly server: Server,
   ) {
     // TODO: if "getting started" page is on the right, replace it
```

Comment:
I looked into this and wasn't able to find a viable solution. There are groups (aka columns) which also have tabs and figuring out which one is the "getting started" page requires iterating though them and active tabs. There also doesn't seem to be an option to replace a tab in a column, it will just open another tab in the column.

Reply:
Not sure what the best option is here.. maybe we should check what other extensions do?

Reply:
We could add a check for the `activeTextEditor` and make sure to open the panel next to it (see [this example](https://github.com/microsoft/vscode-extension-samples/blob/ddae6c0c9ff203b4ed6f6b43bfacdd0834215f83/webview-sample/src/extension.ts#L63-L65)).